content/pt-br/docs/tasks/configure-pod-container/configure-gmsa.md

```diff
@@ -0,0 +1,353 @@
+---
+title: Configurando GMSA Para Pods e Contêineres Windows
+content_type: task
+weight: 20
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.18" state="stable" >}}
+
+Esta página mostra como configurar [Contas de serviço gerenciadas em grupo](https://docs.microsoft.com/en-us/windows-server/security/group-managed-service-accounts/group-managed-service-accounts-overview) (GMSA) 
```

Comment:
Minor: use pt-br instead of en-us
https://learn.microsoft.com/pt-br/windows-server/security/group-managed-service-accounts/group-managed-service-accounts-overview